### PR TITLE
Add pre-commit config

### PR DESCRIPTION
**Pull request by naveen521kk**
As discussed in https://github.com/saadpasta/developerFolio/pull/365#issuecomment-853227996

### CODE REVIEW

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+repos:
```
Comment (kartikcho) [inline .pre-commit-config.yaml:1]: Shouldn't we also add config for CI stuff like [here](https://pre-commit.ci/#configuration) to setup commit msg etc.

**Review by kartikcho — APPROVED** [review]
The config looks good, what's up with pre-commit failing on itself?

File: .pre-commit-config.yaml (hunk 1 above)
Comment (naveen521kk) [reply .pre-commit-config.yaml:1]: I think that isn't required, it uses a default commit message which will make sense.

**Comment by naveen521kk** [comment]
pre-commit.ci autofix

**Pull request merged**